ledger/participant-integration-api/src/test/suite/scala/platform/store/dao/JdbcLedgerDaoOracleAppendOnlySpec.scala

```diff
@@ -12,13 +12,17 @@ class JdbcLedgerDaoOracleAppendOnlySpec
     with JdbcLedgerDaoSuite
     with JdbcLedgerDaoBackendOracleAppendOnly
     with JdbcLedgerDaoPackagesSpec
-    with JdbcLedgerDaoPartiesSpec
-    with JdbcLedgerDaoConfigurationSpec
     with JdbcLedgerDaoActiveContractsSpec
     with JdbcLedgerDaoCompletionsSpec
+    with JdbcLedgerDaoConfigurationSpec
```

Comment:
The ground has shifted since this PR was created, there is now also `JdbcLedgerDaoConfigurationAppendOnlySpec`

Reply:
reconciled with the other AppendOnlySpecs after rebase